### ISSUE TITLE
Can I skip trainning and test with the model you provided in FewX?

### ISSUE THREAD

**Issue by fhong-jpg** [fanq15/FSOD-code#49]
I download the file provide in the page about [FewX](https://github.com/fanq15/FewX):

![图片](https://user-images.githubusercontent.com/54538278/93222365-74d2f880-f7a1-11ea-9406-2b980fe68d3a.png)

will that file work in this situation?

**Comment by fanq15** [comment]
Yes, you can directly test with the model in FewX, but it is trained on COCO, not on FSOD dataset.
